src/d_iwad.cpp

```diff
@@ -677,6 +677,11 @@ int FIWadManager::IdentifyVersion (TArray<FString> &wadfiles, const char *iwad,
 					  "1. Place one or more of these wads in ~/Library/Application Support/" GAMENAMELOWERCASE "/\n"
 					  "2. Edit your ~/Library/Preferences/" GAMENAMELOWERCASE ".ini and add the directories\n"
 					  "of your iwads to the list beneath [IWADSearch.Directories]");
+#elif defined (FLATPAK_BUNDLE)
+					  "1. Place one or more of these wads in ~/.var/app/org.zdoom.GZDoom/.config/" GAMENAMELOWERCASE "/\n"
+					  "2. Edit your ~/.var/app/org.zdoom.GZDoom/.config/" GAMENAMELOWERCASE "/" GAMENAMELOWERCASE ".ini and\n"
```

Comment:
Please convert this 'org.zdoom.GZDoom' string into a string macro, like it is done in `src/version.h` in the section `// More stuff that needs to be different for derivatives.` and apply it here. This way, this can be exported to derivative games without generating new delta.

Reply:
Moved it